static/app/components/events/attachmentViewers/imageViewer.tsx

```diff
@@ -1,30 +1,22 @@
-import styled from '@emotion/styled';
-
 import type {ViewerProps} from 'sentry/components/events/attachmentViewers/utils';
 import {getAttachmentUrl} from 'sentry/components/events/attachmentViewers/utils';
-import PanelItem from 'sentry/components/panels/panelItem';
 
 type Props = Omit<ViewerProps, 'attachment'> & {
   attachment: ViewerProps['attachment'];
   onError?: React.ReactEventHandler<HTMLImageElement>;
   onLoad?: React.ReactEventHandler<HTMLImageElement>;
 };
 
-function ImageViewer({className, onLoad, onError, ...props}: Props) {
+function ImageViewer({onLoad, onError, className, ...props}: Props) {
   return (
-    <Container className={className}>
```

Comment:
Because we use `PanelItem` here, in the modal, the image contains padding and it shouldn't. This component should contain img only element and no wrappers